frontend/src/hooks/hooks.ts

```diff
@@ -1,5 +1,5 @@
 export { useState, useEffect } from 'react';
-export { useLocation, useParams } from 'react-router-dom';
+export { useLocation, useParams, useNavigate } from 'react-router-dom';
```

Comment:
```suggestion
export { useLocation, useParams } from 'react-router-dom';
```
not sure that you use it